pkg/i2gw/providers/kong/converter_test.go

```diff
@@ -131,6 +132,16 @@ func Test_ToGateway(t *testing.T) {
 										Method: ptrTo(gatewayv1beta1.HTTPMethodPost),
 									},
 								},
+								Filters: []gatewayv1beta1.HTTPRouteFilter{
+									{
+										Type: gatewayv1beta1.HTTPRouteFilterExtensionRef,
+										ExtensionRef: &gatewayv1beta1.LocalObjectReference{
+											Group: gatewayv1beta1.Group("configuration.konghq.com/v1"),
+											Kind:  gatewayv1beta1.Kind("KongPlugin"),
```

Comment:
can these use the new consts as well?

Reply:
yep, sorry for missing it. All the references to such an object should be using the const values now 👍